docs/lab/architecture-design.md

```
@@ -333,68 +316,67 @@ graph TD
     - 调用后端 API 获取数据和执行操作。
     - 管理前端状态（如用户登录状态、购物车内容）。
     - **书评组件**：提供书评的展示、发布和交互功能。
-    - **读书笔记组件**：提供读书笔记的创建、编辑和分享功能。
-    - **推荐组件**：展示个性化推荐的图书列表。
+    - **读书笔记组件**：提供读书笔记的创建、编辑和展示功能。
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)